spec/request/v0/facilities/ccp_request_spec.rb

```diff
@@ -4,9 +4,11 @@
 
 RSpec.describe 'Community Care Providers', type: :request, team: :facilities do
   around do |example|
-    VCR.use_cassette('facilities/va/ppms', match_requests_on: %i[path query], allow_playback_repeats: true) do
-      example.run
-    end
+    VCR.insert_cassette('facilities/va/ppms', match_requests_on: %i[path query], allow_playback_repeats: true)
+    VCR.insert_cassette('facilities/va/ppms_new_query', match_requests_on: %i[path query], allow_playback_repeats: true)
```

Comment:
What is the difference between the cassettes `ppms` and `ppms_new_query`?  
Is it that `ppms_new_query` sends lat/long params?  
If so, can we rename it to reflect their difference?  Something like `ppms_lat_long_query`?

Reply:
it's named that way because it's a 'new' query,
I would like the file to go away when I remove the feature flag.